python/whylogs/core/dataset_profile.py

```diff
@@ -48,6 +48,8 @@ def __init__(
         self._columns: Dict[str, ColumnProfile] = dict()
         self._is_active = False
         self._track_count = 0
+        new_cols = schema.get_col_names()
+        self._initialize_new_columns(new_cols)
```

Comment:
shouldn't new_cols be a tuple here too? or is schema.get_col_names doing this already

Reply:
shoot! it returns a tuple, so it's fine

Reply:
`get_col_names()` returns a tuple.